.gitignore

```diff
@@ -3,3 +3,4 @@ composer.lock
 phpunit.xml
 tests/Fixtures/project/var/cache/*
 build/
+.idea
```

Comment:
This should go to your global gitignore. 

See, per example, this: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c but you can find a loot of resources on this topic.

Point here is - rules for ignoring IDE specific files (or system specific files) should not be part of .gitignore files, as we could get into a really large .gitignore files.